src/selectors/entities/preferences.ts

```diff
@@ -89,14 +89,14 @@ export const getVisibleGroupIds = reselect.createSelector(
 export const getTeammateNameDisplaySetting = reselect.createSelector(
```

Comment:
Can you add unit tests to verify the new license-handling behaviour? I suspect some of the tests removed from `makeGetDisplayName` would apply here now instead.

Reply:
Yes you are right! I have added some test cases to cover having no license and license being set to false to ensure that we handle it properly.